### PR TITLE
Change profile name for yourself

### PR DESCRIPTION
**Pull request by LeonardoBein**

### CODE REVIEW

**Comment by LeonardoBein** [comment]
@adiwajshing

**Comment by frmdeveloper** [comment]
> @adiwajshing

At this hour, he is still sleeping or working

**Comment by SanzGantengz** [comment]
> > @adiwajshing
> 
> At this hour, he is still sleeping or working

☕🗿

**Comment by w3nder** [comment]
up

**Comment by ahmadkeren** [comment]
Excuse me, how to apply this changes at my project?
"**npm install https://github.com/adiwajshing/Baileys#1913**"?

**Comment by frmdeveloper** [comment]
> ahmadkeren
> Excuse me, how to apply this changes at my project? "**npm install https://github.com/adiwajshing/Baileys#1913**"?

Silahkan menunggu persetujuan dari pak adiwajsing

File: src/Socket/chats.ts [hunk 1]
```diff
@@ -192,6 +192,10 @@ export const makeChatsSocket = (config: SocketConfig) => {
 		})
 	}
 
+	const updateProfileName = async(name: string) => {
+		return await chatModify({ pushNameSetting: name }, '')
```
Comment (adiwajshing) [inline src/Socket/chats.ts:196]: don't need "return" here

**Pull request merged**